pandas/core/series.py

```diff
@@ -2864,35 +2867,46 @@ def map(self, arg, na_action=None):
         1    foo
         2    bar
         3    baz
+        dtype: object
 
         >>> x.map(y)
         one   foo
         two   bar
         three baz
+        dtype: object
+
+        Map a function to a :class:`pandas.Series`.
 
-        If `arg` is a dictionary, return a new Series with values converted
-        according to the dictionary's mapping:
+        >>> x.map(lambda x: x**2)
+        one      1
+        two      4
+        three    9
+        dtype: int64
+
+        If ``arg`` is a dictionary, return a new :class:`pandas.Series` with
+        values converted according to the dictionary's mapping:
 
         >>> z = {1: 'A', 2: 'B', 3: 'C'}
 
         >>> x.map(z)
```

Comment:
As `z` is not very descriptive, I'd use the dictionary directly as a parameter, and avoid saving it to a variable.